Extension/src/LanguageServer/configurations.ts

```diff
@@ -2005,7 +2002,27 @@ export class CppProperties {
             if (!resolvedPath) {
                 continue;
             }
+
+            // Cache the original value of each path to include any variables or path delimiting.
+            const originalPath: string = curPath;
+            const expandedPaths = this.resolveAndSplit([curPath], undefined, this.ExtendedEnvironment, true, false);
             let pathExists: boolean = true;
+            const incorrectExpandedPaths: string[] = [];
+
+            if (expandedPaths.length > 1) {
```

Comment:
Can you generalize the algorithm so that you don't have to do something special for multiple paths vs. single paths?  I would think you could deal with them the same way (single path is just an array with length == 1).

Reply:
This algorithm should be the same regardless of the number of paths, thanks for pointing this out.